shell/platform/tizen/touch_event_handler.cc

```diff
@@ -76,36 +76,45 @@ Eina_Bool TouchEventHandler::OnTouch(void* data, int type, void* event) {
   if (type == ECORE_EVENT_MOUSE_BUTTON_DOWN) {
```

Comment:
I think it would be better if you replace the if-else with a switch statement.

Reply:
:ok_hand:

Reply:
I got following error with switch statment:
```
../../flutter/shell/platform/tizen/touch_event_handler.cc:97:10: error: case value is not a constant expression
    case ECORE_EVENT_MOUSE_MOVE:
```
I now understand why this is an if-else statement. :smile: